src/Components/JobListing/SimilarJobs/JobSkills/JobSkills.jsx

```diff
@@ -0,0 +1,12 @@
+function JobSkills ({skill}) {
+    return (
+        <>
+        <button className="bg-cyan-500 mr-4 rounded text-white px-2">{skill.skill}</button>
+
+
+        </>
```

Comment:
Check the extra white space, also no need for the <> </> here since you are only returning the button

Reply:
Also, you shouldn't really be using a button for this - the skills boxes on this page are never going to be interactive.